src/poetry/core/utils/toml_file.py

```diff
@@ -15,4 +16,4 @@ def __new__(cls, *args: Any, **kwargs: Any) -> TOMLFile:
             category=DeprecationWarning,
             stacklevel=2,
         )
-        return super().__new__(cls)
+        return super().__new__(cls)  # type: ignore
```

Comment:
See comment above about adding error code to `# type: ignore`

Reply:
ack